crd-generator/src/main/java/io/strimzi/crdgenerator/CrdGenerator.java

```diff
@@ -280,13 +281,15 @@ public CrdGenerator(VersionRange<KubeVersion> targetKubeVersions, ApiVersion crd
      * @param storageVersion If not null, override the storageVersion to the given value.
      * @param servedVersions If not null, override the serverd versions according to the given range.
      * @param conversionStrategy The conversion strategy.
+     * @param describeVersions The API versions to which descriptions should be added
      */
     public CrdGenerator(VersionRange<KubeVersion> targetKubeVersions, ApiVersion crdApiVersion,
                  ObjectMapper mapper, Map<String, String> labels, Reporter reporter,
                  List<ApiVersion> apiVersions,
                  ApiVersion storageVersion,
                  VersionRange<ApiVersion> servedVersions,
-                 ConversionStrategy conversionStrategy) {
+                 ConversionStrategy conversionStrategy,
+                 List<ApiVersion> describeVersions) {
```

Comment:
Does it need to be a list? Our could it be a `VersionRange`? `v1beta1+` saves having to update  the endpoint for `v1beta3`

Reply:
Ok, I changed it to use range instead of list.